miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditRunController.java

```diff
@@ -205,12 +207,22 @@ public ModelAndView setupForm(Run run, PlatformType platformType, ModelMap model
         model.put("title", "New Run");
         model.put("multiplexed", false);
         model.put("metrics", "[]");
+        model.put("partitionNames", "[]");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "partitionNames" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)